app/Console/Command/ProcessDuplicatePlaces.php

```diff
@@ -158,9 +158,11 @@ public function execute(InputInterface $input, OutputInterface $output): int
             }
         }
 
-        if (! $onlySetCanonical) {
+        if (!$onlySetCanonical) {
             // 4. Trigger a SAPI3 reindex on the places removed from duplicate_places
-            $this->reindexPlaces($this->getDuplicatePlacesRemovedFromCluster(), $output, $dryRun);
+
+            $this->reindexPlaces($this->duplicatePlaceRemovedFromClusterRepository->getDuplicatePlacesRemovedFromCluster(), $output, $dryRun);
+            $this->duplicatePlaceRemovedFromClusterRepository->truncateTable();
```

Comment:
Maybe it is better to truncate `duplicate_places_removed_from_cluster` before adding new ones? So we have a longer "history" of the previous run.